test/CadlRanchProjects/type/property/optional/src/Generated/Models/DatetimeProperty.Serialization.cs

```diff
@@ -38,14 +31,13 @@ internal static DatetimeProperty DeserializeDatetimeProperty(JsonElement element
             {
                 return null;
             }
-            Optional<DateTimeOffset?> property = default;
+            Optional<DateTimeOffset> property = default;
             foreach (var property0 in element.EnumerateObject())
             {
                 if (property0.NameEquals("property"u8))
                 {
                     if (property0.Value.ValueKind == JsonValueKind.Null || property0.Value.ValueKind == JsonValueKind.String && property0.Value.GetString().Length == 0)
                     {
-                        property = null;
```

Comment:
After remove implicitly assigning property = null,  The property value will be `DateTimeOffset.MinValue`, not null, is it expected?

Reply:
No this is expected. The value of this property is still null.
Let us sync offline and I will show you how this thing works.